serverless-plugin-canary-deployments.js

```diff
@@ -207,6 +209,21 @@ class ServerlessCanaryDeployments {
     return getMappingsForFunction(this.compiledTpl.Resources);
   }
 
+  getSnsSubscriptionsFor(functionName) {
+    const isEventSourceMapping = (value) => {
+      const match = _.matchesProperty('Type', 'AWS::SNS::Subscription')(value);
+      return match;
+    };
+    const isSubscriptionForFunction = (value) => {
+      const match = _.matchesProperty('Properties.Endpoint.Fn::GetAtt[0]', functionName)(value);
+      return match;
+    };
+    const getMappingsForFunction = _.pipe(
+      _.pickBy(isEventSourceMapping),
+      _.pickBy(isSubscriptionForFunction)
+    );
+    return getMappingsForFunction(this.compiledTpl.Resources);
+  }
```

Comment:
Could you add an extra line break after this function?

Reply:
added extra lines